Common/Mapper/MappingValidator.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
```

Comment:
it's okay to merge it. Since it's not used people won't be confused. August-integration might get cleaned up and we'll loose the changes.